tests/conformance_dsl/mod.rs

```diff
@@ -306,3 +306,26 @@ pub(crate) fn parse_text_exp<'a, I: IntoIterator<Item=&'a Element>>(elems: I) ->
     let val_string = bytes.iter().map(|v| unsafe { String::from_utf8_unchecked(v.to_vec()) }).collect();
     Ok(val_string)
 }
+
+/// Parses absent symbol notation from a symbol within a Toplevel fragment, or produces
+/// Continuation. The notation is '#$<id>' for an absent symbol id, or '#$<name>#<id>' for a symbol
```

Comment:
I thought `#$<id>` was the syntax for a symbol ID to be evaluated within the context being created instead of the current one. Is that what you mean by "absent"?

Reply:
Yea, that's what I mean, that it's absent from the test document's context.